modules/drivers/bigquery-cloud-sdk/test/metabase/driver/bigquery_cloud_sdk_test.clj

```diff
@@ -294,15 +292,65 @@
                         (format "CREATE TABLE %s (transaction_id INT64);"
                                 (fmt-table-name "not_partitioned"))]]
              (bigquery.tx/execute! sql))
-
            (sync/sync-database! (mt/db) {:scan :schema})
-           (is (= table-name->is-filter-required?
-                  (t2/select-fn->fn :name :database_require_filter :model/Table
-                                    :name [:in (keys table-name->is-filter-required?)])))
+
+           (testing "tables that require a filter are correctly identified"
+             (is (= table-name->is-filter-required?
+                    (t2/select-fn->fn :name :database_require_filter :model/Table
+                                      :name [:in (keys table-name->is-filter-required?)]))))
+
+           (testing "partitioned fields are correctly identified"
+             (is (= {["not_partitioned"                 "transaction_id"]   false
+                     ["partition_by_range_not_required" "customer_id"]      true
+                     ["partition_by_range_not_required" "date1"]            false
+                     ["partition_by_range"              "customer_id"]      true
+                     ["partition_by_range"              "date1"]            false
+                     ["partition_by_ingestion_time"     "transaction_id"]   false
+                     ["partition_by_ingestion_time"     "_PARTITIONTIME"]   true
```

Comment:
Do you need to add a test for `_PARTITIONDATE` too? ([docs](https://cloud.google.com/bigquery/docs/querying-partitioned-tables#query_an_ingestion-time_partitioned_table))

Reply:
I think we don't need to sync `_PARTITIONDATE` column, see https://github.com/metabase/metabase/pull/36892/files#r1432189416